lib/src/presentations/sign_up/screen/sign_up_view.dart

```diff
@@ -0,0 +1,207 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:jobspot/src/core/common/custom_toast.dart';
+import 'package:jobspot/src/core/config/localization/app_local.dart';
+import 'package:jobspot/src/core/config/router/app_router.gr.dart';
+import 'package:jobspot/src/core/function/loading_animation.dart';
+import 'package:jobspot/src/core/function/on_will_pop.dart';
+import 'package:jobspot/src/core/resources/data_state.dart';
+import 'package:jobspot/src/presentations/sign_up/cubit/sign_up_cubit.dart';
+import 'package:jobspot/src/presentations/sign_up/widgets/applicant_tab.dart';
+import 'package:jobspot/src/presentations/sign_up/widgets/bussiness_tab.dart';
+import 'package:jobspot/src/presentations/sign_up/widgets/slide_transition_x.dart';
+
+import '../../../core/constants/constants.dart';
+
+class SignUpView extends StatefulWidget {
+  const SignUpView({super.key});
+
+  @override
+  State<SignUpView> createState() => _SignUpViewState();
+}
+
+class _SignUpViewState extends State<SignUpView>
+    with SingleTickerProviderStateMixin {
+  late TabController tabController;
```

Comment:
put "TabController tabController" in SignUpState

Reply:
I fixed it